### PR TITLE
feat: refactor 4626 batch contract

### PR DESCRIPTION
**Pull request by 0xqd**
Minor refactor on the 4626 batch contract.

### CODE REVIEW

File: contracts/src/GetERC4626VaultDataBatchRequest.sol [hunk 1]
```diff
@@ -51,39 +51,33 @@ contract GetERC4626VaultDataBatchRequest {
...
+
             VaultData memory vaultData;
 
             // Get tokens
             vaultData.vaultToken = vaultAddress;
-            vaultData.assetToken = IERC4626Vault(vaultAddress).asset();
+            vaultData.assetToken = assetToken;
 
-            // Check that assetToken exists
             if (codeSizeIsZero(vaultData.assetToken)) continue;
```
Comment (0xKitsune) [inline contracts/src/GetERC4626VaultDataBatchRequest.sol:75]: Lets remove this second codesize zero check.

**Review by 0xKitsune — CHANGES_REQUESTED** [review]

**Comment by 0xKitsune** [comment]
Left a couple comments, otherwise LGTM, thanks for the PR!

File: contracts/src/GetERC4626VaultDataBatchRequest.sol (hunk 1 above)
Comment (0xKitsune) [inline contracts/src/GetERC4626VaultDataBatchRequest.sol:75]: Looks like this is checked above already, can we remove the second check?

**Comment by 0xKitsune** [comment]
Left one minor comment, otherwise LGTM. Thanks for the PR!

File: contracts/src/GetERC4626VaultDataBatchRequest.sol (hunk 1 above)
Comment (0xqd) [reply contracts/src/GetERC4626VaultDataBatchRequest.sol]: hey there, nice catch.

**Review by 0xOsiris — APPROVED** [review]
LGTM, Thanks for the PR!

**Pull request merged**